src/legacy/core_plugins/metric_vis/public/metric_vis_fn.js

```diff
@@ -32,21 +33,118 @@ export const metric = () => ({
     defaultMessage: 'Metric visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    percentage: {
+      types: ['boolean'],
+      default: false,
+      help: i18n.translate('metricVis.function.percentage.help', {
+        defaultMessage: 'Shows metric in percentage mode. Requires colorRange to be set.'
+      })
+    },
+    colorSchema: {
+      types: ['string'],
+      default: '"Green to Red"',
+      options: Object.values(vislibColorMaps).map(value => value.id),
+      help: i18n.translate('metricVis.function.colorSchema.help', {
+        defaultMessage: 'Color schema to use'
+      })
+    },
+    colorMode: {
+      types: ['string'],
+      default: '"None"',
+      options: ['None', 'Label', 'Background'],
+      help: i18n.translate('metricVis.function.colorMode.help', {
+        defaultMessage: 'Which part of metric to color'
+      })
+    },
+    colorRange: {
+      types: ['string'],
+      default: `'[{ "from": 0, "to": 10000 }]'`,
+      help: i18n.translate('metricVis.function.colorRange.help', {
+        defaultMessage: 'Color ranges: array of objects with from and to property.'
```

Comment:
nit: you're one of maybe 6 people that understands the vis object. The rest of us need more guidance than this... what do these numbers mean/do?

Reply:
creates color range. for example passing in an array: 
```
[
 { from: 0, to: 100 },
 { from: 100, to: 200 },
]
```

will color metric in the "first color" (in scheme) if its value is between 0-100 and in second color if value is between 100 and 200

Reply:
That's more helpful than "array of objects with from and to property", and that makes it sound like this should both be part of a new function and be a multi arg (`multi: true`).